adhocteam/Head-Start-TTADP
# Cloud.gov worker process

**Description of change**

* Consolidate worker code back into `src` directory
* Update to cf-cli 7 to take advantage of `processes` definition in manifest.yml
* Add hses import test files to .cfignore to ensure we're not importing them instead of the proper files

**How to test**

Everything should work the same as before the change:

1) `yarn docker:start` 
2) Upload a file to a new AR, and ensure that the worker picks up the file and scans it for viruses.

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/204

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
